docs/doc/dev/doc-generation/how-to.adoc

```diff
@@ -1,28 +1,29 @@
 = Generate rs-server technical documentation
+:sectnums:
 
 This procedure is run automatically in the CI by the 'generate documentation' workflow.
 It can be executed locally to verify the generated documentation (before publishing it for example).
 
-See ${include("description.adoc")} for more details on the process and technical stuff.
+See ${cross_document_ref("description.adoc")} for more details on the process and technical stuff.
 
-. Pre-requises
+== Prerequisites
 
 The local developer environment has been setup.
 The global python project is setup.
 The python sub-project for doc generation is also setup.
 
-. Generate the technical documentation
+== Generate the technical documentation
 
 From the rs-server/docs folder, execute the command:
 [source, bash, indent=0]
 ----
-poetry run asciidoxy --base-dir doc --build-dir ../dist/doc/ --multipage doc/index.adoc
+poetry run asciidoxy --warnings-are-errors --base-dir doc --build-dir ../dist/doc/ --multipage doc/index.adoc
```

Comment:
We also need the ` --image-dir images` here ?

Reply:
absolutely! good catch